modules/openxr/openxr_api.cpp

```diff
@@ -1835,7 +1777,7 @@ XrHandTrackerEXT OpenXRAPI::get_hand_tracker(int p_hand_index) {
 
 Size2 OpenXRAPI::get_recommended_target_size() {
 	RenderingServer *rendering_server = RenderingServer::get_singleton();
-	ERR_FAIL_NULL_V(view_configuration_views, Size2());
+	ERR_FAIL_COND_V(view_configuration_views.size() == 0, Size2());
```

Comment:
nit: this could be `ERR_FAIL_COND_V(view_configuration_views.is_empty(), Size2())`